Backend/app/controllers/event_bookings_controller.rb

```diff
@@ -2,32 +2,35 @@
 
 class EventBookingsController < ApplicationController
   before_action :set_event_booking, only: %i[show update destroy]
-  before_action :set_user, only: %i[index create]
+  before_action :set_user, only: %i[current_event_bookings create]
+  before_action :set_event, only: %i[create ]
+
 
   # GET /event_bookings
-  def index
```

Comment:
why not use the default actions? Is there another any other form of this list that we need?

Reply:
I need to add a method, which returns the list of students who booked that event. In my plans, that method will be named by index :D